tools/deploy.py

```diff
@@ -25,9 +25,14 @@ def parse_args():
     parser.add_argument('deploy_cfg', help='deploy config path')
     parser.add_argument('model_cfg', help='model config path')
     parser.add_argument('checkpoint', help='model checkpoint path')
-    parser.add_argument('img', help='image used to convert model model')
     parser.add_argument(
```

Comment:
I am not sure if multi images is a good idea when conversion models. And idea @RunningLeon ?

Reply:
I don't think this is a good idea.